uploader.json

```diff
@@ -2009,6 +2009,20 @@
                 "Duplicate iptables rule detected"
             ],
             "symbolic_name": "neutron_l3_agent_log"
+        },
+	{
+            "file": "/var/log/neutron/metadata-agent.log",
+            "pattern": [
+                "The resource could not be found"
+            ],
```

Comment:
@ppandey-redhat this pattern will be removed and replaced with the filters from rules when this PR is merged into production.  If this is the specific filter that you want then you will need to change any rule that that adds the filter to be this specific string.

Reply:
@bfahr 
**'Unauthorized: {"error": {"message": "The resource could not be found.", "code": 404, "title": "Not Found"}}'** is the filter which is being used in the rules. It's fine if the above mentioned pattern is removed and replaced with the filters from rules when this PR is merged into production.
The specific filter which is required is **'Unauthorized: {"error": {"message": "The resource could not be found.", "code": 404, "title": "Not Found"}}'**.  Next time onwards, I'll make sure to keep the pattern same as the the filter being used in a rule.